client/src/JobPaginatorState.test.js

```diff
@@ -1,20 +1,10 @@
 import { beforeEach, describe, expect, it, jest } from "@jest/globals";
 import * as JobPaginatorState from "./JobPaginatorState";
-import AjaxHelper from "./AjaxHelper";
-jest.mock("./AjaxHelper");
 
 describe("JobPaginatorState", () => {
-    let ajax;
```

Comment:
Since `findNewPagePosition` has effectively been made public, should a test for it be added here?

Reply:
Made change.